trappy/plotter/ILinePlotGen.py

```diff
@@ -34,6 +34,18 @@
 from IPython.display import display, HTML
 
 
+def to_dygraph_colors(color_map):
```

Comment:
I'm not sure if you wanted it to be part of the ColorMap class or just that the function should be in `ColorMap.py` . I've done the latter because it's too much boilerplate to convert the list of numbers into matplotlib's ScalarMappable only to be converted back into a list of strings with numbers.